pkg/aggregator/aggregator.go

```diff
@@ -92,6 +92,10 @@ var (
 	aggregatorServiceCheck            = expvar.Int{}
 	aggregatorEvent                   = expvar.Int{}
 	aggregatorHostnameUpdate          = expvar.Int{}
+
+	// Hold series to be added to aggregated series on each flush
+	extraSeries     metrics.Series
```

Comment:
nit: it could be nice to have a more descriptive name than `extraSeries`... Maybe `recurrentSeries`?